src/icons/Exclamation.tsx

```diff
@@ -0,0 +1,45 @@
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
+
+const svgElementClasses = classnames(
+  stroke('stroke-formal-accent'),
+  strokeWidth('stroke-1')
+)
+
+export default function () {
+  return (
+    <svg
+      width="13"
```

Comment:
Let's remove `width` and `height` from here and wrap the `svg` into `div` with width and height defined

Reply:
I don't think it's necessary to do it right now because we would rescale it as I see it according to the design. If it's essential, I may come up few ideas